_integration/testsuite/install-contour-working.sh

```diff
@@ -35,42 +35,53 @@ readonly REPO=$(cd ${HERE}/../.. && pwd)
 # registry.
 readonly TAGS="main latest working"
 
-kind::cluster::exists() {
-    ${KIND} get clusters | grep -q "$1"
-}
-
-kind::cluster::load() {
-    ${KIND} load docker-image \
-        --name "${CLUSTERNAME}" \
-        "$@"
-}
+#kind::cluster::exists() {
+#    ${KIND} get clusters | grep -q "$1"
+#}
+#
+#kind::cluster::load() {
+#    ${KIND} load docker-image \
+#        --name "${CLUSTERNAME}" \
+#        "$@"
+#}
+#
+#if ! kind::cluster::exists "$CLUSTERNAME" ; then
+#    echo "cluster $CLUSTERNAME does not exist"
+#    exit 2
+#fi
+#
+## Build the current version of Contour.
+#make -C ${REPO} container IMAGE=docker.io/projectcontour/contour VERSION="v$$"
+#
+#for t in $TAGS ; do
+#    docker tag \
+#        docker.io/projectcontour/contour:"v$$" \
+#        docker.io/projectcontour/contour:$t
+#done
+#
+## Push the Contour build image into the cluster.
+#for t in $TAGS ; do
+#    kind::cluster::load docker.io/projectcontour/contour:$t
+#done
```

Comment:
uncomment

Reply:
this section of code still needs to be uncommented, unless I'm missing something

Reply:
yeah I'm not doin so hot today. It's uncommented now.